pegasus/routes/dev_routes.rb

```diff
@@ -27,3 +30,12 @@
     }
   )
 end
+
+post '/api/dev/set-last-dtt-green' do
+  forbidden! unless rack_env == :test
+  dont_cache
+  forbidden! unless params[:token] == CDO.slack_set_last_dtt_green_token
+
+  sha = GitHub.sha('test')
+  InfraTestTopic.set_green_commit(sha)
```

Comment:
Should this also set `DTT: yes` in the developers topic?  Or is that already handled by the dotd script so we don't want to do it here?